src/OrchardCore.Modules/OrchardCore.Users/JwtClaimTypes.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
```

Comment:
This will need to be removed, or compliant with credits and also checks that the license is compatible.

Reply:
Let's remove that. We already have `OpenIdConnectConstants.Claims` that serves the same purpose.